config/schedule.yml

```diff
@@ -1,4 +1,8 @@
-my_first_job:
-  cron: "*/1 * * * *"
-  class: "HardJob"
+monitor_presence_job_head:
+  cron: "*/15 14-23 * * *"
```

Comment:
Is this 7 days a week, or do we only need this on weekdays?

Reply:
Good catch!